### ISSUE TITLE
Large screen layout: content colspan

### ISSUE THREAD

**Issue by sverps** [scaffold-eth/se-2#184]
The main view is split in 6 cols, but the content is only set to take up 5 cols of it, making it off-center, which feels weird, and leaves some empty space on the right that the content could expand into:

![content](https://user-images.githubusercontent.com/15879327/219867904-f245396b-193a-477c-b78d-9c732048d01e.jpg)

I propose to align it and set the content col-span to 6:

![image](https://user-images.githubusercontent.com/15879327/219868003-e49ab2a1-92f5-421c-819a-3ebadf15986f.png)

**Comment by rin-st** [comment]
Hey! See https://github.com/scaffold-eth/se-2/pull/179#issuecomment-1434939452

**Comment by carletex** [comment]
I think very large inputs or very large areas of text are hard to parse for the eyes.

> The main view is split in 6 cols, but the content is only set to take up 5 cols of it, 

The implementation is a bit weird, yes haha (it was by accident here https://github.com/scaffold-eth/se-2/pull/174).

It's the same effect as if we have a max-width for the main area.

**Comment by carletex** [comment]
Closing it for now, but happy to review when the "core" is finished. We should include Andrea also!

Thanks
